src/hol_extension_context.ts

```diff
@@ -365,12 +379,10 @@ export class HOLExtensionContext {
         }
 
         const selection = editor.selection;
-        const document = editor.document;
         const currentLine = selection.active.line;
-        let text = document.getText(new vscode.Selection(0, 0, currentLine, 0));
-        text = processOpens(text);
+        const newSelection = new vscode.Selection(0, 0, currentLine, 0);
 
-        this.holTerminal!.sendRaw(`${text};\n`);
+        this.sendSelection(editor, newSelection);
```

Comment:
Does this tell HOL4 that we're sending text *starting* at the cursor's line? That doesn't seem right.

Reply:
Looks like the position is just set to (1, 1). Nevermind me. :)